.github/workflows/callable_test-automation.yaml

```
@@ -225,18 +223,17 @@ jobs:
         run: |
           rm -rf /tmp/.buildx-cache
           mv /tmp/.buildx-cache-new /tmp/.buildx-cache
-
   node-image:
     name: Create Node Image
     needs: workflow-setup
     runs-on: ubuntu-20.04
     env:
       GITHUB_REPOSITORY_NAME: ${{ needs.workflow-setup.outputs.GITHUB_REPOSITORY_NAME }}
     steps:
-      - name: Git checkout hyperledger/indy-test-automation
+      - name: Git checkout indy/indy-test-automation
         uses: actions/checkout@v3
         with:
-          repository: hyperledger/indy-test-automation
+          repository: indy/indy-test-automation
```

```suggestion
          repository: hyperledger/indy-test-automation
```
It looks like a find and replace mistake, as it affects all occurrences.